test/scripts/test_private_network.sh

```diff
@@ -35,6 +45,26 @@ ${GOPATH}/bin/goal network stop -r ${NETROOTPATH}
 
 ${GOPATH}/bin/goal network delete -r ${NETROOTPATH}
 
+# Test that genesis generation works correctly
```

Comment:
Can this be removed now that theres an e2e-go test?

Reply:
Reverted in [0ee860d](https://github.com/algorand/go-algorand/pull/5672/commits/0ee860d69f79361077c9b42d83be2b7571a00528)